src/osd/OSDMap.cc

```diff
@@ -3962,11 +3962,13 @@ void OSDMap::print_summary(Formatter *f, ostream& out,
 			   const string& prefix, bool extra) const
 {
   if (f) {
+    f->open_object_section("osdmap");
```

Comment:
this is not correct. normally, we don't print the enclosing "tags" in `print()` or `dump()` function. instead, we do this in the calling sites. see https://github.com/ceph/ceph/blob/626daf4b2043ede719a5d04c70468eb8bf282605/src/mon/Monitor.cc#L2945-L2947.

so i think a better fix is to update `OSDMonitor::preprocess_command()`.